src/dotnet/commands/dotnet-publish/LocalizableStrings.cs

```diff
@@ -29,5 +29,9 @@ internal class LocalizableStrings
         public const string VersionSuffixOption = "VERSION_SUFFIX";
 
         public const string VersionSuffixOptionDescription = "Defines the value for the $(VersionSuffix) property in the project.";
+
+        public const string ProfileProjOption = "profile.proj";
+
+        public const string ProfileProjOptionDescription = "The list of packages to be excluded from publish";
```

Comment:
from publishing perhaps? or are you talking about the command? also missing a period.